package.json

```diff
@@ -40,12 +40,12 @@
     "@babel/register": "^7.9.0",
     "axios-mock-adapter": "^1.18.1",
     "dotenv": "^8.2.0",
-    "eslint": "^8.2.0",
+    "eslint": "^6.8.0",
     "json-loader": "^0.5.7",
     "mime": "^2.4.4",
-    "mocha": "^9.1.3",
+    "mocha": "^6.2.3",
```

Comment:
Why the downgrade? If the older version should be kept, I suggest to pin it, so no-one else updates it again later.

Reply:
@tmathern The library needs to be able to run on older versions of Node, and newer versions of these dependencies didn't work on those older versions. By "pin," does that mean just changing the version to something like `"mocha": "6.2.3"`?

Reply:
Yes, pinning means that you keep that exact version in package.json (up to the Bugfix version).

Reply:
@tmathern I updated the dependency versions - will you please take another look? Thanks!

Reply:
Done.